cas-server-support-oauth/src/test/java/org/apereo/cas/support/oauth/web/OAuth20AccessTokenControllerTests.java

```diff
@@ -146,7 +145,7 @@ public void verifyClientNoClientId() throws Exception {
 
         final MockHttpServletResponse mockResponse = new MockHttpServletResponse();
         requiresAuthenticationInterceptor.preHandle(mockRequest, mockResponse, null);
-        oAuth20AccessTokenController.handleRequest(mockRequest, mockResponse);
+        oAuth20AccessTokenController.handleRequestInternal(mockRequest, mockResponse);
         assertEquals(401, mockResponse.getStatus());
         assertEquals("error=" + OAuthConstants.INVALID_REQUEST, mockResponse.getContentAsString());
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String literal "error=" appears 17 times in this file; the first occurrence is on line 151](https://www.codacy.com/app/mmoayyed/cas/file/2530574633/issues/source?bid=3256115&fileBranchId=3273836#l150)